packages/components/organisms/f-footer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@justeat/f-footer",
-  "version": "4.8.3",
+  "version": "4.9.0",
```

Comment:
I'd argue this should be a major version bump as users will now need to pass `f-vue-icons` as a peer dep?

Reply:
Yeah i wasn't actually sure at first but agree. Will update 👍